datasetinsights/estimators/faster_rcnn.py

```diff
@@ -1039,3 +1101,21 @@ def __call__(self, image, target):
         """transform image to tesnor."""
         image = torchvision.transforms.functional.to_tensor(image)
         return image, target
+
+
+def is_master():
```

Comment:
This should be moved to `estimator.torch_distributed.py` as this actually applies to all estimators training under distributed pytorch.

Reply:
I can move it, but this was done to specifically encapsulate the dependency in this module.